open-source-software-submissions/editor-in-chief-guide.md

```diff
@@ -1,19 +1,131 @@
 # Editor in Chief Guide
 
-## Editor in Chief Role & Responsibilities
+## Editor in Chief role & responsibilities
 
-The Editor in Chief serves for 3 months or a time agreed to by all members of the editorial board. The Editor in Chief fulfills the following roles:
+The Editor in Chief (EiC) is a rotating position that serves 
+for 3 to 6 months or a time agreed to by all members of the editorial 
+board. 
 
-- Watches all issues posted to the software-review repo.
-- Assigns package submissions to other editors, including self, to handle. For the most part, this position rotates among pyOpenSci editors. However, in some cases, an editor may be assigned if the Editor in Chief thinks someone is particularly well-suited to the contents of a package, or if an editor is not able to support the review process at that time due to time or other conflicts.
-- Raises scope/overlap issue with all editors if they see an ambiguous case. This may also be done by handling editors (see below). To initiate discussion, this is posted to the pyOpenSci Slack software review channel, tagging all editors.
-- Responds to pre-submission inquiries and `meta` issues posted to the software-review and software-review-meta repos, similarly pinging channel if discussion is needed. Any editor should feel free to step in on these. See [this section](#responding-to-out-of-scope-submissions) about how to respond to out-of-scope (pre-) submissions.
-- Responds to referrals from JOSS or other publication partners.
-- Monitors the review process pace and reminds other editors to move packages along as needed.
+If the EiC needs to step down at any time before the agreed-upon 
+duration of their tenure, pyOpenSci requests and appreciates 2 weeks notice
+to support finding a new EiC.
 
-## Editor in Chief Checklist
-- Tag issue with `1/editor-checks` tag and assign a main editor.
+### The Editor in Chief fulfills the following roles:
+
+- Watches all issues posted to the software-review repo (either subscribe to repo notifications on GitHub, or watch the `#software-review-feed` channel on Slack).
+- Performs an initial set of editor checks on new package submissions (see template for these checks below)
+- Raises scope/overlap issue with all editors if they see an ambiguous submission. 
+This may also be done by handling editors if appropriate (see note below). If 
+the scope of a package is in question, the EiC should post to the pyOpenSci 
+Slack `software-review` channel, tagging all editors.
+- Assigns package submissions to other editors (including themselves if they have bandwidth), to handle. Assigning may be based upon domain background or just who has bandwidth to lead the review. 
```

Comment:
```suggestion
- Assigns package submissions to other editors to handle. Assigning may be based upon domain background or just who has bandwidth to lead the review. The editor in chief can assign packages to themself for review.
```

Reply:
Just trying to not interrupt a sentence in the middle

Reply:
excellent! thanks!